src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1529,6 +1462,96 @@ private void setMode(Mode m, String msg, boolean log)
             logger.debug(logMsg);
     }
 
+    @VisibleForTesting
+    public Collection<InetAddressAndPort> prepareForBootstrap(long schemaDelay)
```

Comment:
for me: I walked line by line against the branch this got refactored from and looks identical (minus schemaDelay renamed from delay).